lib/index.ts

```diff
@@ -55,6 +55,13 @@ export interface DeboredOptions {
   readonly resources?: ResourceRequirements;
 }
 
+// TODO: Need different approach if we support multiple ingress controllers in the future
+export enum IngressType {
+  SERVICE_LOADBALANCER = 'LoadBalancer',
```

Comment:
To avoid problems, in this case I would not assign string values here and add a `switch` statement below to render the serviceType explicitly. (see below)